apps/game_backend/priv/repo/migrations/20240524202403_rename_dailyquest_to_userquest.exs

```diff
@@ -0,0 +1,7 @@
+defmodule GameBackend.Repo.Migrations.RenameDailyquestToUserquest do
+  use Ecto.Migration
+
+  def change do
+    rename table(:daily_quest), to: table(:user_quest)
```

Comment:
Table names should be in plural, it stores several UserQuests, not just one.
Change to :user_quests.

Reply:
Changed